tests/Tests/HbFaceTest.cs

```diff
@@ -1,26 +1,24 @@
 ﻿using System.IO;
 
 using HarfBuzzSharp;
-
 using Xunit;
 
 namespace SkiaSharp.Tests
 {
 	public class HbFaceTest : SKTest
 	{
-		private static readonly Blob s_blob;
+		public static readonly Blob Blob;
```

Comment:
I'm thinking it might be better to create a new class, such as `HbUtils` so that other tests don't directly reference this test class.

Reply:
Y that makes sense. Had this also in mind. This was also introduced to avoid parallel access to font files.